public/components/marketing-headers/4-dark.html

```diff
@@ -0,0 +1,88 @@
+              fill="currentColor"
+            />
+          </svg>
+        </a>
+      </div>
+
+      <div class="md:flex md:items-center md:gap-12">
+        <nav aria-label="Global" class="hidden md:block">
+          <ul class="flex items-center gap-6 text-sm">
+            <li>
+              <a
+                class="text-gray-500 transition hover:text-gray-500/75 dark:text-white dark:hover:text-white/75"
+                href="#"
+              >
+                About
+              </a>
+            </li>
+
+            <li>
+              <a
+                class="text-gray-500 transition hover:text-gray-500/75 dark:text-white dark:hover:text-white/75"
+                href="#"
+              >
+                Careers
+              </a>
+            </li>
+
+            <li>
+              <a
+                class="text-gray-500 transition hover:text-gray-500/75 dark:text-white dark:hover:text-white/75"
+                href="#"
+              >
+                History
+              </a>
+            </li>
+
+            <li>
+              <a
+                class="text-gray-500 transition hover:text-gray-500/75 dark:text-white dark:hover:text-white/75"
+                href="#"
+              >
+                Services
+              </a>
+            </li>
+
+            <li>
+              <a
+                class="text-gray-500 transition hover:text-gray-500/75 dark:text-white dark:hover:text-white/75"
+                href="#"
+              >
+                Projects
+              </a>
+            </li>
+
+            <li>
+              <a
+                class="text-gray-500 transition hover:text-gray-500/75 dark:text-white dark:hover:text-white/75"
+                href="#"
+              >
+                Blog
+              </a>
+            </li>
+          </ul>
+        </nav>
+
+          <div class="block">
+            <button
+              type="button"
+              title="Your dashboard"
+              class="overflow-hidden rounded-full border-2 border-teal-600 bg-transparent shadow-md"
+            >
+              <img src="https://i.pravatar.cc/300?img=1" alt="Your dashboard" class="h-10 w-10" />
```

Comment:
`size-10` and can we use an image from Unsplash, please?

Reply:
No problem, I will send back fixes soon